packages/ruby_debug.rb

```diff
@@ -0,0 +1,63 @@
+# Adapted from Arch Linux ruby-rubocop PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=ruby-rubocop
+
+require 'package'
+
+class Ruby_debug < Package
+  description 'Ruby debug.rb'
+  homepage 'https://github.com/ruby/debug'
+  version '1.6.2'
+  compatibility 'all'
+  source_url 'SKIP'
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_debug/1.6.2_armv7l/ruby_debug-1.6.2-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_debug/1.6.2_armv7l/ruby_debug-1.6.2-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_debug/1.6.2_i686/ruby_debug-1.6.2-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_debug/1.6.2_x86_64/ruby_debug-1.6.2-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '2bdb5f3152576c078f7375bf83cf87c19a620b91c80e6363d127e5aa68ec790c',
+     armv7l: '2bdb5f3152576c078f7375bf83cf87c19a620b91c80e6363d127e5aa68ec790c',
+       i686: '0046afda40659300d9237be3c45ea444f43c1ccadfbc82e787d4528fb55d98c3',
+     x86_64: '449811904d7d9b2969504f5930d3347a4e1590bda94dea9b43d5856935a7a93e'
+  })
+
+  no_fhs
+
+  depends_on 'libyaml'
+  depends_on 'ruby'
+  # depends_on 'xdg_base'
+
+  # @xdg_config_home = ENV.fetch('XDG_CONFIG_HOME', nil)
+  # @xdg_config_home = "#{CREW_PREFIX}/.config" if @xdg_config_home.to_s.empty?
+
+  def self.build
+    # system 'bundle install'
+    # system 'rake build'
+  end
+
+  def self.install
+    # system "gem install --build=#{CREW_DEST_DIR} pkg/rubocop-#{version}.gem"
+    FileUtils.mkdir_p CREW_DEST_PREFIX
+  end
+
+  def self.postinstall
+    @gem_name = name.sub('ruby_', '')
+    system "gem uninstall -Dx --force --abort-on-dependent #{@gem_name} || true"
+    system "gem install -N #{@gem_name} --conservative"
```

Comment:
Simplify to:
```
  system "gem uninstall -Dx --force --abort-on-dependent debug || true"
  system "gem install -N debug --conservative"
 ```
?

Reply:
The rationale is to use a template for as many of these packages as possible.

Reply:
Fair enough.

Reply:
While testing in the Docker x86_64 container, I encountered the following:
```
$ gem install -N debug --conservative
Fetching debug-1.6.2.gem
Fetching reline-0.3.1.gem
Successfully installed reline-0.3.1
Building native extensions. This could take a while...
ERROR:  Error installing debug:
	ERROR: Failed to build gem native extension.

    current directory: /usr/local/lib64/ruby/gems/3.1.0/gems/debug-1.6.2/ext/debug
/usr/local/bin/ruby -I /usr/local/lib64/ruby/site_ruby/3.1.0 extconf.rb
creating Makefile

current directory: /usr/local/lib64/ruby/gems/3.1.0/gems/debug-1.6.2/ext/debug
make DESTDIR\= sitearchdir\=./.gem.20221014-10938-ewkazp sitelibdir\=./.gem.20221014-10938-ewkazp clean

current directory: /usr/local/lib64/ruby/gems/3.1.0/gems/debug-1.6.2/ext/debug
make DESTDIR\= sitearchdir\=./.gem.20221014-10938-ewkazp sitelibdir\=./.gem.20221014-10938-ewkazp
compiling debug.c
compiling iseq_collector.c
linking shared-object debug/debug.so

current directory: /usr/local/lib64/ruby/gems/3.1.0/gems/debug-1.6.2/ext/debug
make DESTDIR\= sitearchdir\=./.gem.20221014-10938-ewkazp sitelibdir\=./.gem.20221014-10938-ewkazp install
make: /usr/local/bin/mkdir: No such file or directory
make: *** [Makefile:203: .sitearchdir.-.debug.time] Error 127

make install failed, exit code 2

Gem files will remain installed in /usr/local/lib64/ruby/gems/3.1.0/gems/debug-1.6.2 for inspection.
Results logged to /usr/local/lib64/ruby/gems/3.1.0/extensions/x86_64-linux/3.1.0/debug-1.6.2/gem_make.out
```

Reply:
What is your `PATH`?

Reply:
Does this work?
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=ruby_debug CREW_TESTING=1 crew update
crew install ruby_debug
```

Reply:
I see the same issue.

Trying to rebuild ruby w/o coreutils installed to see if that makes a difference.

Reply:
Make sure `uutils_coreutils` is not installed either.

Reply:
> The rationale is to use a template for as many of these packages as possible.

#6306 :eyes:

Reply:
@saltedcoffii: I don't see a template for installing ruby gems in that pull request.  Should we add?